consumer.go

```diff
@@ -359,21 +362,29 @@ func (child *partitionConsumer) dispatcher() {
 	close(child.feeder)
 }
 
+func (child *partitionConsumer) preferedBroker() (*Broker, error) {
+	if child.replicaInited {
+		broker, err := child.consumer.client.Broker(child.preferredReadReplica)
+		if err == nil {
+			return broker, nil
+		}
```

Comment:
does the error not important? could we log it?

Reply:
Not important at all..but could log it.